### PR TITLE
[synthetics] Update PL gov doc to mention FIPS-compliant requests

### PR DESCRIPTION
**Pull request by ruizb**
### What does this PR do?
Update the documentation about Synthetics Private Location running on GovCloud, to mention the requests that are FIPS compliant.

### Motivation
The next Private Location release will contain a component that allows some of its requests to run in FIPS mode, for PLs whose site is ddog-gov.com.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/private_locations/_index.md [hunk 1]
```diff
@@ -97,9 +97,9 @@ To pull test configurations and push test results, the private location worker n
...
-| 443  | `intake.synthetics.ddog-gov.com` | Used by the private location to pull test configurations and push test results to Datadog using an in-house protocol based on [AWS Signature Version 4 protocol][1]. |
+| Port | Endpoint                         | Description                                                                                                                                                                                                                     |
+|------|----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 443  | `intake.synthetics.ddog-gov.com` | Used by the private location to pull test configurations and push test results to Datadog using an in-house protocol based on [AWS Signature Version 4 protocol][1]. For versions >= 1.32.0, these requests are FIPS compliant. |
```
Comment (buraizu) [inline content/en/synthetics/private_locations/_index.md:102]:
```suggestion
| 443  | `intake.synthetics.ddog-gov.com` | Used by the private location to pull test configurations and push test results to Datadog using an in-house protocol based on [AWS Signature Version 4 protocol][1]. For versions 1.32.0 and later, these requests are FIPS compliant. |
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, I just suggested a minor update for accessibility. Also, could you remove your changes to `content/fr/synthetics/private_locations/_index.md`? It's sufficient to make changes to the English language repo; this initiates the translation process for the `/fr` and `/ja` documentation repos. If you'd like to know more about the process, feel free to reach out internally, and I can share some additional resources which explain it in more detail.

File: content/en/synthetics/private_locations/_index.md [hunk 2]
```diff
@@ -97,9 +97,9 @@ To pull test configurations and push test results, the private location worker n
...
-| 443  | `intake.synthetics.ddog-gov.com` | Used by the private location to pull test configurations and push test results to Datadog using an in-house protocol based on [AWS Signature Version 4 protocol][1]. |
+| Port | Endpoint                         | Description                                                                                                                                                                                                                            |
+|------|----------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 443  | `intake.synthetics.ddog-gov.com` | Used by the private location to pull test configurations and push test results to Datadog using an in-house protocol based on [AWS Signature Version 4 protocol][1]. For versions 1.32.0 and later, these requests are FIPS compliant. |
```
Comment (buraizu) [inline content/en/synthetics/private_locations/_index.md:102]:
```suggestion
| 443  | `intake.synthetics.ddog-gov.com` | Used by the private location to pull test configurations and push test results to Datadog using an in-house protocol based on [AWS Signature Version 4 protocol][1]. For versions 1.32.0 and later, these requests are Federal Information Processing Standards (FIPS) compliant. |
```

**Review by buraizu — APPROVED** [review]
Thanks for making that change, just added a final suggestion as I noticed FIPS is not spelled out elsewhere in the doc.

**Comment by ruizb** [comment]
Hello @buraizu, thank you for your reviews! Feel free to merge the PR as It seems I am not authorized to do it.

**Review by m-rousse — APPROVED** [review]

**Pull request merged**